go/cmd/dolt/commands/dump.go

```diff
@@ -167,7 +167,16 @@ func (cmd DumpCmd) Exec(ctx context.Context, commandStr string, args []string, d
 			return HandleVErrAndExitCode(err, usage)
 		}
 
-		err2 := addBulkLoadingParadigms(dEnv, fPath)
+		dbName, err2 := getActiveDatabaseName(ctx, dEnv)
```

Comment:
It would be a little more idiomatic to reuse `err` instead of creating `err2`. If you change this var name to `err` it'll "shadow" the previous `err` variable – that's a pretty common pattern you'll see in the codebase.